go.mod

```diff
@@ -59,10 +59,7 @@ require github.com/olekukonko/tablewriter v0.0.5
 
 require github.com/google/uuid v1.3.0
 
-require (
-	github.com/chzyer/readline v1.5.1 // indirect
-	github.com/zitadel/oidc v1.13.4
```

Comment:
Note that we also maintain some private extension repos, including an OIDC implementation that is not yet ready for release. Unfortunately, there seems to be no other way than adding their deps to the main `go.mod` file. So if you update `go.mod` and `go.sum` without the additional code being in the project directory, those deps will be removed even though they may be needed. Ideally, we will find another solution to this in the future.